tests/test_lithium.py

```diff
@@ -235,6 +278,141 @@ def check_result(r, i):
                 raise
 
 
+class InterestingnessTests(TestCase):
+    cat_cmd = [sys.executable, "-c", ("import sys;"
+                                      "[sys.stdout.write(f.read())"
+                                      " for f in"
+                                      "     ([open(a) for a in sys.argv[1:]] or"
+                                      "      [sys.stdin])"
+                                      "]")]
+    ls_cmd = [sys.executable, "-c", ("import glob,itertools,os,sys;"
+                                     "[sys.stdout.write(p+'\\n')"
+                                     " for p in"
+                                     "     (itertools.chain.from_iterable(glob.glob(d) for d in sys.argv[1:])"
+                                     "      if len(sys.argv) > 1"
+                                     "      else os.listdir('.'))"
+                                     "]")]
+    sleep_cmd = [sys.executable, "-c", "import time;time.sleep(3)"]
```

Comment:
Can we make sleep_cmd accept a variable timeout number, other than 3? I'm not really insistent on this though.